src/test/java/core/basesyntax/impl/ReadFileFromCsvTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.ReadFileService;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReadFileFromCsvTest {
+    private static final String VALID_FILE_PATH = "src/test/resources/test_input.csv";
+    private static final String INVALID_FILE_TO_READ = "invalid_file.csv";
+    private static ReadFileService fileReader;
+
+    @BeforeAll
+    static void setUp() {
+        fileReader = new ReadFileFromCsv();
+    }
+
+    @Test
+    public void read_ValidFile_Ok() {
+        List<String> lines = fileReader.readFromFile(VALID_FILE_PATH);
+        Assertions.assertNotNull(lines);
+        assertEquals(3, lines.size());
```

Comment:
Is it enough to test that size is as expected maybe it makes sence to compare content as well